src/utils.ts

```diff
@@ -554,6 +555,20 @@ function isStreamReset(e: Error): number | false {
   }
 }
 
+/**
+ * Increases the total number of registered event handlers before a node warning is emitted.
+ * In most cases this is not needed but in the case where you have one event emitter for multiple handlers you'll need
+ * to increase the limit.
+ * @param target - The specific `EventTarget` or `EventEmitter` to increase the warning for.
+ * @param limit - The limit before the warning is emitted, defaults to 100000.
+ */
+function setMaxListeners(
+  target: EventTarget | NodeJS.EventEmitter,
+  limit: number = 100000,
```

Comment:
This should be in the module configuration defaults.

Do not hardcode the constant here.

Propagate the config variable all the way up to PK system config.

Ensure consistent naming across all evented push-pull modules.

@brynblack @tegefaulkes @amydevs